policyengine_us/parameters/gov/states/ct/tax/income/credits/property_tax/threshold/agi.yaml

```diff
@@ -0,0 +1,26 @@
+description: Connecticut limits its Property Tax Credit to filers with federal adjusted gross income below the following amount, based on filing status.
+metadata:
+  label: Connecticut Property Tax Credit income threshold
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Connecticut General Statutes, Chapter 229, Sec. 12-704c
+    href: https://www.cga.ct.gov/current/pub/chap_229.htm#sec_12-704c
+  - title: 2022 Form CT-1040 Connecticut Resident Income Tax Return Instructions
+    href: https://portal.ct.gov/-/media/DRS/Forms/2022/Income/2022-CT-1040-Instructions_1222.pdf#page=26
+  - title: 2021 Form CT-1040 Connecticut Resident Income Tax Return Instructions
+    href: https://portal.ct.gov/-/media/DRS/Forms/2021/Income/CT-1040-Online-Booklet_1221.pdf#page=30 
+
+
+SINGLE:
+  2021-01-01: 49_500
```

Comment:
is this when the credit get reduced? If so, we can use as the start variable in reduction